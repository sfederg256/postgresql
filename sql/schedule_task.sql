CREATE OR REPLACE FUNCTION test_notice() RETURNS void AS $$
BEGIN
    RAISE NOTICE 'Hello from task!';
END;
$$ LANGUAGE plpgsql;

INSERT INTO scheduler.scheduled_tasks (name, sql_command, next_run_at, repeat_interval)
VALUES ('Test Task', 'SELECT test_notice();', now(), '1 minute');

SELECT pg_sleep(5);

SELECT status, last_run_at FROM scheduler.scheduled_tasks WHERE name = 'Test Task';