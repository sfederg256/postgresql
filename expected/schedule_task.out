CREATE FUNCTION
INSERT 0 1
 pg_sleep 
----------
 
(1 row)

 status  | last_run_at 
---------+-------------
 pending | 
(1 row)

